emailit/api.py

```diff
@@ -32,6 +32,8 @@ def construct_mail(recipients=None, context=None, template_base='emailit/email',
     language = language or translation.get_language()
     with force_language(language):
         recipients = recipients or []
+        if not isinstance(recipients, list):
```

Comment:
I don't think this will work for other iterables (e.g tuple). How about checking for basestring instead and assuming it is an iterable otherwise?